src/content/ui-api/management-api/authentication/index.md

```diff
@@ -123,11 +123,87 @@ Let's go into details for the most relevant properties (also called "claims") of
 
 ## Authenticating for programmatic access {#service-auth}
 
-For programmatic access, for example from CI/CD pipelines, you should not rely on the above authentication mechanism. Instead, please use [service accounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/).
+For programmatic access, for example from CI/CD pipelines, you should not rely on the above authentication mechanism. Depending on your use case, you may follow one of the below scenarios.
 
-Each Giant Swarm installation provides a service account named `automation` in the `default` namespace.
+### Remote access to the Management Cluster {#remote-mc}
 
-**Note:** This service account comes with a quite powerful set permissions. **We strongly recommend to create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+Each Giant Swarm installation provides a service account named `automation` in the `default` namespace that may be used for creating a self-contained `kubeconfig` file. **Note however**, this account comes with a quite powerful set of permissions, thus **we strongly recommend to create a specific service account for each application**, binding it to specific roles granting only the required permissions in the required namespaces.
+
+Regardless of service account you decide to use, and assuming you are currently logged in to the right installation, you can utilize the below instruction to create a config file.
```

Comment:
```suggestion
Regardless of which service account you decide to use. From the management cluster, you can utilize the step below to create a config file.
```

I think this can be shorter. The important part IMO is that they are no the MC not the WC.

Reply:
agree!